package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "schema2typebox",
-  "version": "1.4.1",
+  "version": "1.5.0",
```

Comment:
I go back and forth on this, but typically do this as a part of the `yarn publish` workflow on main; obviously no issue at this point, just a thought.

Reply:
Yeah I am not sold on that one either.

Will keep a todo thinking about a good publishing workflow in the future. For now the focus is on your issues/features! :]